adafruit_platformdetect/constants/boards.py

```diff
@@ -250,6 +252,7 @@
     BEAGLEBONE_POCKETBEAGLE: (("A2", "A335PBGL00A2"),),
     BEAGLEBONE_GREEN: (("1A", "A335BNLT...."), ("UNKNOWN", "A335BNLTBBG1")),
     BEAGLEBONE_GREEN_WIRELESS: (("W1A", "A335BNLTGW1A"),),
+    BEAGLEBONE_GREEN_GATEWAY: (("A335BNLTGG1A"),),
```

Comment:
this should be a tuple like the ones above?

Reply:
Yeah, My bad. I've made the correction to BEAGLEBONE_GREEN_GATEWAY: (("G1A", "A335BNLTGG1A"),),